emperor/support_files/js/view.js

```diff
@@ -93,20 +118,30 @@ function DecompositionView(decomp, asPointCloud) {
     writable: false
   });
 
-  // setup this.markers and this.lines
-  if (this.usesPointCloud) {
+  this._initGeometry();
+}
+
+DecompositionView.prototype._initGeometry = function() {
+  this.oldMarkers = this.markers;
+  if (this.oldMarkers.length > 0)
+    this.needsSwapMarkers = true;
+  this.markers = [];
+
+  //TODO FIXME HACK:  Do we need to swap lines as well?
```

Comment:
Can you elaborate on what you mean in this comment?

Reply:
I don't have a biplot dataset (that's where lines come from right?), so I'm not sure what the behavior is or should be when there are lines in the scene.  I think I went with disabling lines and re-enabling lines, but presumably the parallel plot view could have lines in it at some point that would require new swapping logic.

Reply:
Yep that should be the right answer. Can you remove this comment line then?